### DISCUSSION TITLE
This is an error message while trying to build mega2560 and mega16u2 firmware.

### DISCUSSION THREAD

**Discussion by js-midi** [shanteacontrols/OpenDeck#111] [Q&A]
This is an error message while trying to build mega2560 and mega16u2 firmware.
I want to know if my computer is the problem
please review again

![스크린샷(1)](https://user-images.githubusercontent.com/80445117/136380544-4074212f-60b1-4763-aa31-6f7fa391110a.png)

**Comment by paradajz** [comment]
Please stop creating duplicate topics.
